src/App.js

```diff
@@ -29,30 +29,41 @@ class App extends Component {
       this.getActivePlayers(allPlayers)
     }
   }
-  
+
+  showFrontOrBack = (sideToShow) => {
+      this.setState({ showCardSide: sideToShow})
+  }
+
+  setCurrentPlayer = (chosenPlayer) => {
+    this.setState({ chosenPlayer: chosenPlayer })
+  }
+
   getActivePlayers(players) {
     const activePlayers = players.filter(player => player.Status === 'Active')
     this.setState({ activePlayers: activePlayers })
   }
 
-  setPlayersName() {
-    this.state.activePlayers.map(player => {
-      const fullName = `${player.LastName}, ${player.FirstName}`
-      player.name = fullName
-      return fullName
-    })
-  }
-
   render() {
     return (
       <div className="app">
         <header className="header">WHO'S THAT CUB?</header>
-        <Find players={this.state.activePlayers} setPlayersName={this.setPlayersName}/>
+          <Find 
+            players={this.state.activePlayers} 
+            setCurrentPlayer={this.setCurrentPlayer} 
+            showFrontOrBack={this.showFrontOrBack}
+            showCardSide={this.state.showCardSide}
+            chosenPlayer={this.state.chosenPlayer}
+          />
         <Switch>
-          <Route exact path="/" render={() => <CardFront playerData={favoritePlayer} />} /> 
-          <Route path="/:{playerName}" /> if :playerName, render cardfront of that player
-          <Route path="/:{playerName}/back" /> :playerName/back, render cardback of that player
-          <Route path="/*" /> same as "/"
+          <Route exact path="/" render={() => <CardFront 
+            chosenPlayer={this.state.chosenPlayer} 
+            showFrontOrBack={this.showFrontOrBack}
+            showCardSide={this.state.showCardSide}/> } />
+          <Route path="/back" render={() => <CardBack 
+            chosenPlayer={this.state.chosenPlayer} 
+            showFrontOrBack={this.showFrontOrBack} 
+            showCardSide={this.state.showCardSide}/> } />
```

Comment:
Both the front and back components need access to the chosen player, and the showFrontOrBack and showCardSide might get refactored.